tequilapi/endpoints/proposals.go

```diff
@@ -123,17 +129,54 @@ func NewProposalsEndpoint(mc server.Client) *proposalsEndpoint {
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (pe *proposalsEndpoint) List(resp http.ResponseWriter, req *http.Request, params httprouter.Params) {
 	providerID := req.URL.Query().Get("providerId")
+	fetchQuality := req.URL.Query().Get("fetchQuality")
 	proposals, err := pe.mysteriumClient.FindProposals(providerID)
 	if err != nil {
 		utils.SendError(resp, err, http.StatusInternalServerError)
 		return
 	}
-	proposalsRes := proposalsRes{mapProposalsToRes(proposals, proposalToRes)}
+
+	addQualityToRes := func(p proposalRes) proposalRes { return p }
+	if fetchQuality == "true" {
+		addQualityToRes, err = addQuality(pe.mysteriumMorqaClient)
```

Comment:
Quality oracle should be absolutely optional feature.
It means, if it's offline, proposal listing is still possible.